Localize/lang/strings.json

```
@@ -782,6 +780,7 @@
   "_+JCU+Y.comment": "Tooltip for on-prem gateway connection checkbox",
   "_+W2vTp.comment": "Placeholder description for a newly inserted Email parameter",
   "_+W9fkq.comment": "Timezone value ",
+  "_+ZSBrq.comment": "Accessibility label",
   "_+dlOgj.comment": "Timezone value ",
```

Trailing whitespace?

```suggestion
  "_+dlOgj.comment": "Timezone value",
```

---

Yeah it seems like so